src/main/java/org/fundacionjala/coding/marco/Examen.java

```diff
@@ -0,0 +1,16 @@
+package org.fundacionjala.coding.marco;
+
+/**
+ * This was made by Marco Mendez September 15, 2017.
+ */
+public class Examen {
+    /**
+     * This method calculate digital root.
+     *
+     * @param value test.
+     * @return test.
+     */
+    public int root(int value) {
+        return (value != 0 && value % 9 == 0) ? 9 : value % 9;
```

Comment:
this may refactored like this.
```java
    public int root(int value) {
        int module = value % 9;
        return module == 0 ? 9 : module;
    }
```